src/toast/redux/toast.js

```diff
@@ -0,0 +1,21 @@
+import { OPEN_TOAST, CLOSE_TOAST, REMOVE_TOAST } from './toast.actions'
+
+export const initState = []
+export const initAction = { type: 'UNKNOWN' }
+
+export default (state = initState, { type, payload } = initAction) => {
+  switch (type) {
+    case OPEN_TOAST:
+      return [...state, { print: true, ...payload }]
+    case CLOSE_TOAST: {
+      return state.map((item, idx) => {
+        if (idx === 0) return { ...item, print: false }
+        return item
+      })
+    }
+    case REMOVE_TOAST: {
```

Comment:
What is the purpose of `CLOSE_TOAST` ? (as a developper, since `REMOVE_TOAST` remove it from DOM too right ?)

Reply:
this action allows you to apply a css animation to hidde toast , Perhaps action name HIDE_TOAST is better ?

Reply:
Ok, but I don't get what's the difference between `CLOSE_TOAST` and `HIDE_TOAST` ?